### DISCUSSION TITLE
How can i get the Radio Inline except for the label

### DISCUSSION THREAD

**Discussion by bomshteyn** [filamentphp/filament#8165] [Help]
I am trying to get a radio input to have label above while the options should be inline.

When using the `->inline()` method the label becomes inline as well. 

Is there a way to achieve this?

View with inline: 
<img width="460" alt="Current View" src="https://github.com/filamentphp/filament/assets/4259699/c9a8b8b3-1f4f-4e4b-bffc-2cf8c22310a3">

View Without Inline:
<img width="453" alt="Without Inline:" src="https://github.com/filamentphp/filament/assets/4259699/6fd212e9-b51a-4b37-90c0-190d370e5384">

What we would like to have:
<img width="459" alt="What we would like to have" src="https://github.com/filamentphp/filament/assets/4259699/3b22e648-3be7-4392-a415-1364e001e45e">

**Comment by pboivin** [comment]
I don't think there's an option for this but it should be pretty straightforward with some custom CSS. Combined with `->inline()`, you can target the `label` element with the following:
```css
    flex-direction: column;
    align-items: start;
```
